src/main/scala/com/amazon/deequ/analyzers/QuantileNonSample.scala

```diff
@@ -0,0 +1,307 @@
+package com.amazon.deequ.analyzers
+
+import scala.collection.immutable.ListMap
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.ClassTag
+import scala.util.control.Breaks._
+
+
+class QuantileNonSample[T](val sketchSize: Int,
```

Comment:
Please format according to our [coding conventions](https://github.com/databricks/scala-style-guide)

Reply:
Done. Reformat the locations of both variables and extends.